database/seedCreation.js

```diff
@@ -1,17 +1,17 @@
 const { LoremIpsum } = require('lorem-ipsum');
```

Comment:
also remove this library (`npm remove`) and don't require it if you don't need it